thirdparty/ocl/CMakeLists.txt

```diff
@@ -2,6 +2,36 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+function(get_lib_name TARGET_NAME LIBRARY_NAME)
+    if (CMAKE_BUILD_TYPE STREQUAL "Debug")
+        set(BUILD_SUFFIX ${IE_DEBUG_POSTFIX})
+    else()
+        set(BUILD_SUFFIX ${IE_RELEASE_POSTFIX})
+    endif()
```

Comment:
can we use `IE_BUILD_POSTFIX` which is already build type independent?

Reply:
We can't use this variable since in cmake configuration time it will be something like `$<$<CONFIG:Debug>:d>$<$<CONFIG:Release>:>` for Windows

Reply:
got it, thanks